components/02-elements/rating/_rating.scss

```diff
@@ -0,0 +1,78 @@
+$rating-font-size: $font-size-large;
+$rating-active   : $orange;
+
+.rating {
+    unicode-bidi: bidi-override;
+    direction: rtl;
+    font-size: $rating-font-size;
+    cursor: pointer;
+    text-align: left;
+
+    span {
+        display: inline-block;
+        position: relative;
+
+        &:hover {
+            &:before {
+                content: "\2605";
+                position: absolute;
+                color: $rating-active;
+            }
+
+            ~ span {
+                &:before {
+                    content: "\2605";
+                    position: absolute;
+                    color: $rating-active;
+                }
+            }
+        }
+    }
+
+    &--static {
+        position: relative;
```

Comment:
just curious: why you use modifier here? inside this modifier there are element not styled in default module... What is your point?

Reply:
you mean `rating--static__fill` class? It's only related to this variant so I don't style it in default module

Reply:
make `&--static` an element not a modifier

Reply:
done